src/main/java/com/bloomtechlabs/coderheroesbea/controllers/ProfilesController.java

```diff
@@ -0,0 +1,64 @@
+package com.bloomtechlabs.coderheroesbea.controllers;
+
+import com.bloomtechlabs.coderheroesbea.entities.Profiles;
+import com.bloomtechlabs.coderheroesbea.services.ProfilesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import javax.validation.Valid;
+import java.net.URI;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/profile/")
+public class ProfilesController {
+
+    private ProfilesService service;
+
+    public void ProfilesController(ProfilesService service) {
+        this.service = service;
+    }
+
+
+
+    @GetMapping
+    public ResponseEntity<List<Profiles>> getAllProfiles() {
+        List<Profiles> profiles = service.findAll();
+        return ResponseEntity.ok().body(profiles);
+    }
+
+    @GetMapping(value = "/{profiles_id}")
+    public ResponseEntity<Profiles> getProfileByID(@PathVariable("profiles_id") Long profile_id) {
+        Optional<Profiles> profile = service.find(profile_id);
+        return ResponseEntity.of(profile);
+    }
```

Comment:
You might want to consider the case of what happens when you can't find a profile. Is there some sort of message you want to send back to the front end saying, "User doesn't exist".